src/calculateRentalCost.js

```diff
@@ -29,6 +29,20 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const basicRate = 40;
+  const basePrice = basicRate * days;
+  const optimalDiscount = 50;
+  const basicDiscount = 20;
```

Comment:
optimal is very subjective. Just name them large and small discount. And it will be the final commit :)